integration-cli/docker_api_containers_test.go

```diff
@@ -1730,3 +1736,48 @@ func (s *DockerSuite) TestContainersApiCreateMountsCreate(c *check.C) {
 		}
 	}
 }
+
+func (s *DockerSuite) TestContainersApiCreateMountsTmpfs(c *check.C) {
```

Comment:
It seems there is some project-local naming convention that encourages `Test.*Api.*`?
If not, perhaps we need to fix that in separate PR?

```console
$ grep "func (.*Suite) Test.*Api" integration-cli/*.go | wc -l
158
$ grep "func (.*Suite) Test.*API" integration-cli/*.go | wc -l
2
```